softlayer/vm/pool/vm_pool_manager_test.go

```diff
@@ -0,0 +1,53 @@
+package vm_pool_test
+
+import (
+	"errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/maximilien/bosh-softlayer-cpi/softlayer/vm/pool"
+
+	fakes "github.com/maximilien/bosh-softlayer-cpi/softlayer/vm/pool/fakes"
+
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+)
+
+var _ = Describe("VMInfoDB", func() {
+	var (
+		vmInfoDB VMInfoDB
+		fakeDB   *fakes.FakeDB
+		logger   boshlog.Logger
+		err      error
+	)
+
+	BeforeEach(func() {
+		logger = boshlog.NewLogger(boshlog.LevelDebug)
+
+		fakeDB = fakes.NewFakeDB()
+	})
+
+	//TODO: add tests for methods: #QueryVMInfobyAgentID, #QueryVMInfobyID, #DeleteVMFromVMDB, #InsertVMInfo, #UpdateVMInfoByID
```

Comment:
They are already addressed in integration test for concurrency issue. Removed the comment.